src/panels/config/devices/device-detail/ha-device-card.ts

```diff
@@ -81,6 +84,18 @@ export class HaDeviceCard extends LitElement {
               </div>
             `
           : ""}
+        ${this.domains.includes("mqtt")
```

Comment:
Let's not add this in `ha-device-card` but instead, make a separate card specific for mqtt options.